ui_framework/components/modal/confirm_modal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { KuiModal } from '../modal';
+import { KuiModalFooter } from './modal_footer';
+import { KuiModalHeader } from './modal_header';
+import { KuiModalBody } from './modal_body';
+import { KuiModalBodyText } from './modal_body_text';
+import { KuiButton } from '../index';
+
+export function KuiConfirmModal({ message, title, onCancel, onConfirm, cancelButtonText, confirmButtonText, ...rest }) {
+  return (
+    <KuiModal style={{ 'width': '450px' }} { ...rest }>
+      {
+        title ?
+          <KuiModalHeader>
+            <div className="kuiModalHeader__title" data-test-subj="confirmModalTitleText">
```

Comment:
Yea.  Just to confirm, you are opposed to refactoring these div wrappers that have just a className?  There are so many of them and creating a separate file + test for each one creates so much noise.